packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.tsx

```diff
@@ -1,11 +1,17 @@
+import { css } from '../../../Utilities';
 import { BaseButton } from '../BaseButton';
-import './DefaultButton.scss';
-import '../ButtonCore/ButtonCore.scss';
+import styles from './DefaultButton.scss';
 
 export class DefaultButton extends BaseButton {
-  protected _variantClassName = 'ms-Button--default';
+  protected classNames = {
```

Comment:
I'm also unclear on why `classNames` is used instead of passing strings directly as usual. Is this to make it possible to customize class names with a single prop, rather than multiple class name props as we have now?